addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,30 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object (runs on server only via module framework).
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+ #include "script_component.hpp"
+
+ params ["_logic"];
+
+ private _object = attachedTo _logic;
+ if (isNull _object) then {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,displayTextStructured);
+ } else {
+    if (isMultiplayer) then {
+        _object enableSimulationGlobal !(simulationEnabled _object);
+    } else {
+        _object enableSimulation !(simulationEnabled _object);
+    };
```

Comment:
I think this whole then block can be simplified to:
`_object enableSimulationGlobal !(simulationEnabled _object);`
`enableSimulation` and `enableSimulationGlobal` are synonymous in SP as the SP client is always the server.